src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -119,7 +121,13 @@ def generate_document_context(contract_definition=None, user=None, order=None):
         user_phone_number = user.phone_number
 
     if order:
-        organization_logo = image_to_base64(order.organization.logo)
+        logo_checksum = file_checksum(order.organization.logo)
+        (logo_image, _created) = DocumentImage.objects.get_or_create(
+            checksum=logo_checksum,
+            defaults={"file": order.organization.logo},
+        )
+        organization_logo_id = str(logo_image.id)
+
```

Comment:
On certificate, I added an `images` field.
So when I add a new DocumentImage on the resource, I link this image to the resource through this relation. In this way, it's easy to track unused images to clear database later. Maybe it's overengineer.

https://github.com/openfun/joanie/blob/e1385530353c110ae147c6efd49d9ebed0c838a0/src/backend/joanie/core/models/certifications.py#L100-L106